modules/rosa-cluster-wide-proxy.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * networking/proxies/configuring-cluster-wide-proxy.adoc
+
+[id="rosa-cluster-wide-proxy-config_{context}"]
+= Configuring a cluster-wide proxy using {product-title} (ROSA)
+
+You can add a proxy to your cluster when you are creating a new one. Prior to installation, however, you should verify that the proxy is accessible from the intended cluster virtual private cloud (VPC) and its private subnets.
+
+.Procedure
+To create a cluster with a proxy, run the following command:
+[source, terminal]
+----
+rosa create cluster \
```

Comment:
Is "ocm" vs "rosa" in this command literally the only difference between these two modules? I compared visually, and this was the only thing I saw (though I might be missing something).

If this is the case, then I'd consider using a single file for both, and defining a temporary attribute at the top of this file (depending on what the assembly context is, so you know if it's dedicated or rosa), and then using that attribute in the command here. Let me know if you have any questions on how to do that.